src/components/views/elements/AccessibleButton.tsx

```diff
@@ -92,20 +91,23 @@ export interface IAccessibleButtonProps extends React.InputHTMLAttributes<Elemen
  * @param {Object} props  react element properties
  * @returns {Object} rendered react
  */
-export default function AccessibleButton<T extends keyof JSX.IntrinsicElements>({
-    element = "div" as T,
-    onClick,
-    children,
-    kind,
-    disabled,
-    inputRef,
-    className,
-    onKeyDown,
-    onKeyUp,
-    triggerOnMouseDown,
-    ...restProps
-}: IProps<T>): JSX.Element {
+const AccessibleButton = forwardRef(function <T extends keyof JSX.IntrinsicElements>(
```

Comment:
it would be good to write in the doc-comment for `AccessibleButton` that it exposes an optional `ref` prop which it will set to the rendered DOM `element`.

Reply:
This is a fundamental of React https://react.dev/learn/referencing-values-with-refs and described by the type
<img width="563" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/2403652/94f58ce7-3474-4f17-91db-79e190497b61">